oci-ignite-advanced/intro/intro.md

```diff
@@ -0,0 +1,47 @@
+# Introduction
+
+The labs in this workshop cover a level 100 set of skills for **Oracle Cloud Infrastructure (OCI) Core Services**, including Virtual Cloud Networks (VCN), Autonomous Database, Oracle Kubernetes Engine (OKE). These labs also provide some of the foundational skills and knowledge you will need to attempt the [Oracle Cloud Infrastructure Architect Associate Certification](https://education.oracle.com/oracle-cloud-infrastructure-2022-architect-associate/pexam_1Z0-1072-22) offered by Oracle University.
+
+The Oracle Cloud Infrastructure Architect Associate exam is designed for individuals who possess a strong foundational knowledge of architecting infrastructure using Oracle Cloud Infrastructure (OCI) services. The certification is intended for candidates with technical backgrounds who want to validate their understanding of Oracle Cloud Infrastructure.
+
+The labs featured on this page cover various solutions, skill levels, and categories based on Oracle Cloud Infrastructure (OCI). 
+
+Deploy a complete data warehousing platform that scales to both your technical and analytic requirements.
+
+
+Estimated Workshop Time: 1 hour and 30 minutes
+
+### Objectives
+
+In this workshop, you will:
+
+- Get introduced to the regions available on Oracle Cloud
+- Apply the Ignite Basics knowledge
+- Create a Virtual Cloud Network
+- Create an Autonomous Database
+
+**<font color="blue">Optional</font>**:
```

Comment:
We do not recommend having html code in markdown files and coloring of the text. Please remove the optional heading. You have that point part of Objectives and add optional in the braces.